pkgs/os-specific/linux/libratbag/default.nix

```diff
@@ -14,12 +14,13 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [
-    meson ninja pkg-config gitMinimal swig check valgrind
+    meson ninja pkg-config gitMinimal swig check valgrind wrapGAppsHook3
```

Comment:
For the wrapper to find `glib` typelibs, `wrapGAppsNoGuiHook` should be sufficient